admin-dev/themes/new-theme/js/pages/product/components/combination-modal/CombinationModal.vue

```diff
@@ -188,7 +188,7 @@
         editCombinationUrl: '',
         loadingCombinationForm: false,
         submittingCombinationForm: false,
-        combinationList: <JQuery>$(ProductMap.combinations.combinationsListContainer),
+        combinationList: <JQuery>$(ProductMap.combinations.combinationsFormContainer),
```

Comment:
On of the casts we need to remove in the future, it breaks the type and can lead to `is undefined` bugs

Reply:
OK but not in this PR it's not the scope ^^